hack/update-codegen.sh

```diff
@@ -8,11 +8,29 @@ set -o pipefail
 REPO_ROOT=$(git rev-parse --show-toplevel)
 cd "${REPO_ROOT}"
 
-echo "Generating with deepcopy-gen"
 GO111MODULE=on go install k8s.io/code-generator/cmd/deepcopy-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/register-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/conversion-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/client-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/lister-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/informer-gen
+GO111MODULE=on go install k8s.io/code-generator/cmd/openapi-gen
 export GOPATH=$(go env GOPATH | awk -F ':' '{print $1}')
 export PATH=$PATH:$GOPATH/bin
 
+go_path="${REPO_ROOT}/_go"
+cleanup() {
+  rm -rf "${go_path}"
+}
+trap "cleanup" EXIT SIGINT
+
+cleanup
```

Comment:
Why keep this since we already put the `cleanup` in `trap`?

Reply:
Copy it from: 
https://github.com/karmada-io/karmada/blob/2974563c6a14c9fd4b85bacc8ae44ec6013bf99b/hack/verify-codegen.sh#L15-L20

It's cannot be careful.

Reply:
Can you help confirm it? I don't think `cleanup` is needed here.

Reply:
`go_path` maybe contaminated by other programs/scripts, it is safe to clean it before generate files.

Reply:
Yeah, that makes sense to me.